electron-app/src/components/settings/api-keys/ExchangeSettings.vue

```diff
@@ -99,16 +105,22 @@ import BaseExternalLink from '@/components/base/BaseExternalLink.vue';
 import ConfirmDialog from '@/components/dialogs/ConfirmDialog.vue';
 import MessageDialog from '@/components/dialogs/MessageDialog.vue';
 import ExchangeBadge from '@/components/ExchangeBadge.vue';
+import { convertToGeneralSettings } from '@/data/converters';
```

Comment:
the conversion should never happen inside a component, this should probably happen in an action

Reply:
Check my comment here: https://github.com/rotki/rotki/pull/905#discussion_r411754540